wordpress/mod.ts

```diff
@@ -32,178 +126,158 @@ export default function (userOptions?: Partial<Options>) {
 
   return (site: Site) => {
     site.data("wp", wp);
+    site.hooks.addTransformer = wp.addTransformer.bind(wp);
```

Comment:
Please, use a name like `addWordPressTransformer` or similar to make it more clear that is related with this plugin.

Reply:
Avoid name collision, got it.